app/assets/javascripts/discourse/app/models/post-stream.js

```diff
@@ -697,7 +697,10 @@ export default RestModel.extend({
    * */
   triggerNewPostInStream(postId, opts) {
     deprecated(
-      "Please use triggerNewPostsInStream, this method will be removed July 2021"
+      "Please use triggerNewPostsInStream, this method will be removed July 2021",
```

Comment:
What year are we btw?

Reply:
Yeah I know, we have lots of deprecations like this, but we don't currently have the confidence to remove them 😅 

This PR is one step towards improving our tooling and confidence, so we can start removing old stuff like this 🔥